Linux_scripts/rhui-check/README.md

```diff
@@ -0,0 +1,50 @@
+# RHUI Check Script
+
+## Overview
+
+This script validates server configuration and connectivity to the RHUI Servers provided by Azure to the PAYG RHEL images.
+Instead of performing an end-to-end conectivity test, the rhui-check.py performs individual validations of the different components required to have a successful communication 
```

Comment:
I believe there was a similar script for testing. What happened to that one? Is this an addition or replacement of the previous script?

Reply:
Yes, this one is replacing it. Once this one is merged, I'll submit another PR to retire the other one. Instead of modifying the previous one, we decided to work it from the scratch